etna/models/deadline_ma.py

```diff
@@ -301,5 +314,17 @@ def predict(self, ts: TSDataset, prediction_size: int, return_components: bool =
         ts.df = new_df
         return ts
 
+    def params_to_tune(self) -> Dict[str, "BaseDistribution"]:
+        """Get default grid for tuning hyperparameters.
+
+        This grid doesn't tune ``seasonality`` parameter. It expected to be set by the user.
+
+        Returns
+        -------
+        :
+            Grid to tune.
+        """
+        return {"window": IntUniformDistribution(low=1, high=10, step=1)}
```

Comment:
Probably for `MovingAverageModel` we need higher `high` value. I am not really sure about value I set here.

Reply:
Do we need higher value? For yearly seasonality it is high enough I guess, the same for monthly